src/bin/conllx-select.rs

```diff
@@ -0,0 +1,55 @@
+extern crate conllx;
+extern crate conllx_utils;
+extern crate getopts;
+extern crate stdinout;
+
+use std::env::args;
+use std::fs;
+use std::io::BufWriter;
+
+use conllx::WriteSentence;
+use conllx_utils::or_exit;
+use getopts::Options;
+use stdinout::{Input, Output};
+
+fn print_usage(program: &str, opts: Options) {
+    let brief = format!(
+        "Usage: {} [options] ID_FILE [INPUT_FILE] [OUTPUT_FILE]",
+        program
+    );
+    print!("{}", opts.usage(&brief));
+}
+
+fn main() {
+    let args: Vec<String> = args().collect();
+    let program = args[0].clone();
+
+    let mut opts = Options::new();
+    opts.optflag("h", "help", "print this help menu");
+    let matches = or_exit(opts.parse(&args[1..]));
+
+    if matches.opt_present("h") {
+        print_usage(&program, opts);
+        return;
+    }
+
+    if matches.free.is_empty() || matches.free.len() > 3 {
+        print_usage(&program, opts);
+        return;
```

Comment:
The other utilities may do this wrong, but we should use `process::exit(1)` here, since it's an error.